app/posts/find-teacher-training/2022-02-11-updating-international-candidates-section.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Updating the ‘International candidates’ section
+description: Why we have updated and renamed this section
+date: 2022-02-11
+screenshots:
+  items:
+    - text: International candidates - sponsorship may be available
+      src: find-sponsorship-available.png
+    - text: International candidates - sponsorship not available
+      src: find-sponsorship-not-available.png
+related:
+  items:
+    - text: GitHub pull request making the change
+      href: https://github.com/DFE-Digital/find-teacher-training/pull/1138
+    - text: Design history entry for previous change
+      href: /find-teacher-training/new-international-students-section/
+---
+
+In August last year, we [added a new ‘International students’ section](/find-teacher-training/new-international-students-section/) as a place where we could explain things that people applying from outside the UK need to know, including whether the course offered visa sponsorship or not.
```

Comment:
```suggestion
We previously added an [‘International students’ section](/find-teacher-training/new-international-students-section/) to help candidates from outside the UK get the information they need, such as whether visa sponsorship is available for a course.
```

Reply:
I thought 'as a place where we could explain things' didn't sound very user centred haha!

Reply:
Ha, good edit, thanks!